lib/likeno/entity.rb

```diff
@@ -95,11 +95,20 @@ def self.exists?(id)
       request(exists_action, id_params(id), :get)['exists']
     end
 
-    def self.find(id)
-      response = request(find_action, id_params(id), :get)
+    def self.find(id, prefix = '', headers = {})
+      response = request(find_action, id_params(id), :get, prefix, headers)
       new(response[entity_name], true)
     end
 
+    def self.all(headers = {}, params = {})
```

Comment:
Wouldn't it be better to add an `all_action` on the [helper](https://github.com/pgt/likeno/blob/add-possibility-to-send-header/lib/likeno/helpers/crud_request_parameters.rb)? The action may not always be an empty string.

Also, the prefix may not always be empty as well. I can agree the method is most certainly going to be `get` though.